components/ai_chat/core/common/mojom/settings_helper.mojom

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module ai_chat.mojom;
+
+import "brave/components/ai_chat/core/common/mojom/ai_chat.mojom";
+
+struct ModelWithSubtitle {
+  Model model;
+  string subtitle;
+};
+
+interface AIChatSettingsHelper {
```

Comment:
I don't think having Settings in the name is technically the best choice, but I don't really see a good alternative at the moment. It's not important because when we change to AIChatService and that having the main mojom interface then I assume these functions will be incorporated to that cc @yrliou